pkg/util/namespace.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Sees we don't need this file.

Reply:
Sorry, It was to be moved to test/framework, I forgot to delete this file.